perk-store-webapps/src/main/webapp/vue-app/components/perk-store/OrderDetail.vue

```diff
@@ -116,7 +116,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             <div v-if="!refunding && (!order.remainingQuantityToProcess || isError)">
               <v-icon class="green--text me-1" size="16">fa-check-circle</v-icon>{{ $t('exoplatform.perkstore.label.processingDone') }}
             </div>
-            <template v-else-if="userData && userData.canEdit">
+            <template v-else-if="orderCheckIn">
```

Comment:
Please check that, it's a **security problem**, we can change the value of` eXo.env.portal.userName` (from the browser console) to null and therefore we will have the value of **orderCheckIn** always true